### DISCUSSION TITLE
Comentarios en el vivo de las clases

### DISCUSSION THREAD

**Discussion by lopez-juandavid** [Launch-X-Latam/MisionFrontEnd#165] [Q&A] [answered]
**Buen dia explorers** 

Tengo un problema referente a comentar en los espacios en vivo de las clases por team, ya que cada vez que lo hago nunca sale mi comentario. y por otro lado, nunca me salen los comentarios que realizan mis compañeros por team cuando veo las clases por diferido. 

Es algo normal o solamente me pasa a mi?

**Comment by doguedogue** [comment, marked as answer]
Es completamente normal.

De hecho por la forma que está diseñado el en vivo tu puedes escribir tus preguntas o comentarios pero no las ves reflejadas más que en la pantalla del MC.

Trata de no presionar la casilla anónimo cuando escribas tu pregunta o comentario o no lo leerá el MC.

No te desanimes sigue compartiendonos lo que desees 🚀

**Reply by lopez-juandavid** [reply]
Muchas gracias por sacarme de la duda.

**Comment by josmanyel** [comment]
> **Buen dia explorers**
> 
> Tengo un problema referente a comentar en los espacios en vivo de las clases por team, ya que cada vez que lo hago nunca sale mi comentario. y por otro lado, nunca me salen los comentarios que realizan mis compañeros por team cuando veo las clases por diferido.
> 
> Es algo normal o solamente me pasa a mi?

Es difícil que los veas más no imposible. Yo lo he logrado porque expresamente he escrito en momento de control para ver si mi comentario salta a la vista y ... Si sale con un retardo de un segundo o dos más o menos, pero entiendo que cuando todos podemos escribir es casi imposible porque el chat del MC es como una metralla porque somos más de mil personas escribiendo a la vez.

**Comment by Amhedriel** [comment]
> **Buen dia explorers**
> 
> Tengo un problema referente a comentar en los espacios en vivo de las clases por team, ya que cada vez que lo hago nunca sale mi comentario. y por otro lado, nunca me salen los comentarios que realizan mis compañeros por team cuando veo las clases por diferido.
> 
> Es algo normal o solamente me pasa a mi?

Buenas!!!

Te explico cómo va lo del chat, tú tienes 2 opciones en el chat, lo digo de memoria.

1. Comentarios destacados
2. Tus comentarios

Cuando haya un comentario destacado, el Administrador del Live (Mission Commander) puede anclarlo en ese apartado.
Cuando tú eres el que comenta, solo verás tus propios comentarios, es así como funciona.

Te preguntas porque no me aparece el chat como en el live¿?

Simple, el que está haciendo el Live es el Administrador (MC) y él tiene que ver los mensajes que le llegan, en este caso el Administrador nos está compartiendo su pantalla de chat, en el que se encuentra entre los mensajes de los 1500 explorers tus propios mensajes, para que lo explique un poco mejor sería:

1. Tu escribes un "mail".
2. Él MC recibe el "mail".
3. El "mail" se muestra en su pantalla, pero se interpone ya sea arriba o abajo de otros "mails" que escriben los demás viewers.
4. Él MC muestra su pantalla donde le llegan los "mails" y lo comparte con todos.
5. Cómo es un "mail" enviado directamente al MC de tí, los demás participantes no recibirán ese mensaje en sus buzones, pero como lo envían expresamente al MC él si lo verá.
6. Entonces el MC para ver el "mail" le aparece en una pantalla, que él comparte, para que todos podamos ver los "mails" que le están llegando en tiempo real.
7. Si le llega un "mail" que el MC quiera compartir o destacar con todos, entonces aparecerá en el apartado de ***Comentarios destacados***

Como te acabo de explicar, en resumen se diría que no es un chat como tal, sino un servicio de mensajería privada unilateral, el cual nuestro Mission Commander comparte con nosotros.

Espero haber aclarado tus dudas y de alguien más que tenía dudas con esto, saludos.
